commands/eval.py

```diff
@@ -0,0 +1,125 @@
+from typing import List, Optional
+
+import click
+from kili.client import Kili
+
+from commands.common_args import EvaluateOptions, Options
+from kiliautoml.models._base_model import (
+    BaseEvaluateArgs,
+    BaseInitArgs,
+    ModelConditionsRequested,
+)
+from kiliautoml.models.auto_get_model import auto_get_instantiated_model
+from kiliautoml.utils.helpers import (
+    curated_job,
+    get_assets,
+    get_content_input_from_job,
+    get_project,
+    print_evaluation,
+)
+from kiliautoml.utils.logging import logger, set_kili_logging
+from kiliautoml.utils.type import (
+    AssetStatusT,
+    JobNameT,
+    MLBackendT,
+    ModelNameT,
+    ModelRepositoryT,
+    ProjectIdT,
+    VerboseLevelT,
+)
+
+
+@click.command()
+@Options.project_id
+@Options.api_endpoint
+@Options.api_key
+@Options.ml_backend
+@Options.model_name
+@Options.model_repository
+@Options.target_job
+@Options.ignore_job
+@Options.max_assets
+@Options.clear_dataset_cache
+@Options.batch_size
+@Options.verbose
+@EvaluateOptions.asset_status_in
+@EvaluateOptions.model_path
```

Comment:
what's the difference with model name ?

Reply:
EvaluateOptions -> Options ? wdyt ?

Reply:
not possible because of `asset_status_in` which is specific to evaluate